filer/static/filer/js/addons/dropdown-menu.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+/* global django */
```

Comment:
any particular reason to keep this so custom? i think it would be ok to use a custom dropdown implementation that does just dropdown handling, and figure out rest of the cases separately